user_guide_src/source/changelogs/v4.7.0.rst

```diff
@@ -97,6 +97,8 @@ Deprecations
 Bugs Fixed
 **********
 
+- **Cookie:** The ``CookieInterface::SAMESITE_STRICT``, ``CookieInterface::SAMESITE_LAX``, and ``CookieInterface::SAMESITE_NONE`` constants are now written in ucfirst style to be consistent with usage in the rest of the framework.
```

Comment:
this needs to be marked as bc break, since `$samesite` variable is came from config, it can came from `app/` config

https://github.com/codeigniter4/CodeIgniter4/blob/9d032efb2586601679c4da6947615769ee268109/app/Config/Cookie.php#L90

which can be miss type

Reply:
sorry i did not get how this will be a bc break. can you explain more?

Reply:
for example, user wrote:

```
 public string $samesite = 'LAX'; 
```

in `app/Config/Cookie.php`, it previously working as transformed before check, it now not working because direct compare, right?

The change is ok, just add "BC break" note.

Reply:
ah ok. then if I will return the ucfirst-strtolower transform this can pass as non-bc break, right?

Reply:
Thanks, added